Documentation/Concepts/Fluid/Index.rst

```diff
@@ -7,159 +7,178 @@
-developers can implement complex functionality and therefore extend the
-original functionality of Fluid to their heart's content. ViewHelpers are built
-in the programming language PHP. Having said that, website integrators or
-editors are not required to learn or understand these (this is the
-responsibility of a software developer). Integrators only need to **apply**
-them -- and this is as easy as adding an HTML tag such as :html:`<image.../>` to an
-HTML file.
-
-More than 80 ViewHelpers are shipped with the TYPO3 core already. They enable
-integrators and web developers to use translations of variables, generate forms
-and dynamic links, resize images, embed other HTML files and even implement
-logical functions such as :html:`if ... then ... else ...`. An overview of the
-available ViewHelpers and how to apply them can be found in the
-:doc:`Fluid ViewHelper reference <t3viewhelper:Index>`.
+TYPO3 uses a template engine to generate the necessary HTML
+markup for a website. The template engine provides data from
+the backend, such as content or menu structures, which can
+then be formatted with HTML.
+
+TYPO3's template engine of choice is called Fluid. Fluid's
+syntax is inspired by the syntax of HTML or XML documents,
+so it should already be familiar for users that already know
+HTML.
+
+While Fluid is extendable by PHP developers, knowing any
+PHP is not necessary to write templates for TYPO3. However,
+Fluid comes with its own syntax rules, which need to be
+obeyed when writing templates.
+
+..  _fluid-basics:
+
+Fluid Basics
+============
+
+Accessing Variables
+-------------------
+
+An integral part of Fluid are variables. This is the data
+that is provided by the backend to be used inside your
+template. You can access variables like this:
+
+..  code-block:: xml
+
+    <h1>{myHeadline}</h1>
+
+
+If a variable contains subitems, these can be accessed
+with the dot syntax:
+
+..  code-block:: xml
+
+    <p>{myVariable.mySubItem}</p>
+
+Modifying Variables
+-------------------
+
+You can also do some basic math operations:
+
+..  code-block:: xml
+
+    {myNumber + 5}
+
+
+If you need to modify the provided data even more, you
+can use so-called ViewHelpers. These are functions that
+take some input values, perform operations on those
+values and then output the result. The following example
+converts a variable to uppercase:
+
+..  code-block:: xml
+
+    <f:format.case mode="upper">{myText}</f:format.case>
+
+If you want to perform multiple operations on one variable
+or if your templates become more complex, you might also
+want to use :ref:`Fluid's inline notation <t3coreapi:fluid-inline-notation>`.
+
+Using Control Structures
+------------------------
+
+ViewHelpers are also used for so-called control structures.
+If you want to add a condition to your template, you can
+use the `<f:if>` ViewHelper:
+
+..  code-block:: xml
+
+    <f:if condition="{myVariable} == 'hello'">
```

Comment:
Could you add info about compound conditions and else-if too, this is a common "problem" to actually use conditions.
(Or link to f:if viewhelper docs?)